### PR TITLE
python3Packages.twilio: 6.51.1 -> 6.56.0

### PR DESCRIPTION
**Pull request by fabaff**
###### Motivation for this change
Update to latest upstream release 6.56.0

Change log: https://github.com/twilio/twilio-python/releases/tag/6.56.0

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/servers/home-assistant/default.nix [hunk 1]
```diff
@@ -347,6 +347,7 @@ in with py.pkgs; buildPythonApplication rec {
     "uptime"
     "vacuum"
     "verisure"
+    "wake_on_lan"
```
Comment (mweinelt) [inline pkgs/servers/home-assistant/default.nix:350]: Looks unrelated.

Comment (fabaff) [reply pkgs/servers/home-assistant/default.nix:350]: Yes, sorry. Came from another PR.

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 119121` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>python38Packages.twilio</li>
    <li>python39Packages.twilio</li>
  </ul>
</details>

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 119121` at 2df1ab33 run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built successfully:</summary>
  <ul>
    <li>python38Packages.twilio</li>
    <li>python39Packages.twilio</li>
  </ul>
</details>
